onnxruntime/core/providers/cpu/ml/svmclassifier.cc

```diff
@@ -99,24 +117,34 @@ Status SVMClassifier<T>::Compute(OpKernelContext* ctx) const {
   {
     int64_t current_weight_0 = n * stride;
     int64_t maxclass = -1;
-    double maxweight = 0.f;
     std::vector<float> decisions;
     std::vector<float> scores;
     std::vector<float> kernels;
     std::vector<int64_t> votes;
+    float sum;
```

Comment:
initial value = 0 maybe?

Reply:
replaced by double sum, initialization is done in a loop